Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation_README.md

```diff
@@ -74,11 +80,17 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
 | AutoCloseAlert | Whether to close the alert automatically or manually, after an analyst's review. | False | Optional |
 | ShouldRescanBenign | Whether to rescan \(Using WildFire detonate file\) benign files. | True | Optional |
 | ShouldManualReviewFP | Whether to ask for a manual review before false positive handling. Should be True or False | False | Optional |
+| SHA256 | The SHA256 hash of the file to respond to. | alert | Optional |
```

Comment:
note that new inputs where added with default input alert ( without all the DT explanation).

Reply:
Yep, same as the first comment.